business-central-parent/business-monitoring-webapp/pom.xml

```diff
@@ -1390,6 +1390,15 @@
     <finalName>${project.artifactId}</finalName>
     <outputDirectory>${project.build.directory}/${project.build.finalName}/WEB-INF/classes</outputDirectory>
 
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>com.github.spotbugs</groupId>
+          <artifactId>spotbugs-maven-plugin</artifactId>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+
```

Comment:
Why we have this here, isn't it duplicating what is in https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/921 ?